### PR TITLE
Adding new community configuration management policies (cnf)

### PR DESCRIPTION
**Pull request by alosadagrande**
This PR intends to add 4 configuration management policies:

- Install performance addon operator
- Install ptp operator
- Install sr-iov network operator
- Label nodes

This effort is included in the task of automating the installation and configuration of CNF operators with ACM and OpenShift. Detailed information can be found in this repository: https://github.com/alosadagrande/acm-cnf

### CODE REVIEW

File: community/CM-Configuration-Management/policy-label-worker-nodes.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+  name: policy-clustercnf10-tag-workers
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-label-worker-nodes.yaml]: remove hard coded ns

**Comment by ycao56** [comment]
can you resolve the conflicts? we updated the table to use markdown table instead

File: community/CM-Configuration-Management/policy-label-worker-nodes.yaml (hunk 1 above)
Comment (alosadagrande) [reply community/CM-Configuration-Management/policy-label-worker-nodes.yaml]: removed

**Review by ycao56 — APPROVED** [review]

**Comment by dockerymick** [comment]
Will review in the morning

**Review by dockerymick — APPROVED** [review]
Made a very small grammatical update, see my commit. The updates look good to me, thanks!

**Pull request merged**